Darker.DarkKiller.BlazeUi/Pages/Home.razor

```diff
@@ -27,34 +31,109 @@
 
 @if (game.Players.Count is not 0)
 {
-    <FluentDataGrid Items="@game.Players.AsQueryable()">
-        <PropertyColumn Property="@(p => p.DisplayName)" Title="Display Name" Sortable="true" />
-        <PropertyColumn Property="@(p => p.Order)" Title="Throw Order" Sortable="true" />
-    </FluentDataGrid>
+    <FluentCard>
+        <FluentDataGrid Items="@game.Players.AsQueryable()">
+            <PropertyColumn Property="@(p => p.DisplayName)" Title="Display Name" Sortable="false" />
+            <PropertyColumn Property="@(p => p.Order)" Title="Throw Order" Sortable="true" IsDefaultSortColumn="true" InitialSortDirection="SortDirection.Ascending" />
+            <TemplateColumn Align="Align.Center" Title="Options" Sortable="false">
+                @{
+                    var isFirst = game.Players.IndexOf(context) is 0;
+                    var isLast = game.Players.IndexOf(context) == game.Players.Count - 1;
+                }
+                <FluentButton OnClick="() => ChangePlayerEditing(context)" aria-label="Edit player" IconEnd="@(new Icons.Regular.Size16.Edit())" />
+                <FluentButton Disabled="isFirst" OnClick="() => ShiftThrowPositionUp(context)" aria-label="Move player up" IconEnd="@(new Icons.Regular.Size16.ArrowUp())" />
```

Comment:
Move the 'sort order moving' to the sort order column
This will involve making the Sort Order a template column, look up to how still enable sorting on this column